src/libraries/Microsoft.PowerFx.Core/Entities/External/IExternalDataSource.cs

```diff
@@ -19,6 +19,9 @@ internal interface IExternalDataSource : IExternalEntity, IExternalPageableSymbo
 
         bool RequiresAsync { get; }
 
+        // !!! Change this in PA
+        bool IsClearable { get; }
```

Comment:
adding Breaking Internals tag for this.

Reply:
Im removing this for now. I'll add it back when implementing Clear/ClearCollect functions. I'll also remove the breaking internals label.